### ISSUE TITLE
docker mongo fails

### ISSUE THREAD

**Issue by HearMeWhisper** [lowercasename/gathio#158]
gathio-1        | Welcome to gathio! The app is now running on http://localhost:3000
gathio-1        | Nodemailer is ready to send emails.
gathio-1        | Error saving log entry!
gathio-1        | /app/node_modules/.pnpm/mongoose@5.13.22/node_modules/mongoose/lib/drivers/node-mongodb-native/collection.js:198
gathio-1        |           const err = new MongooseError(message);
gathio-1        |                       ^
gathio-1        | 
gathio-1        | MongooseError: Operation `events.find()` buffering timed out after 10000ms
gathio-1        |     at Timeout.<anonymous> (/app/node_modules/.pnpm/mongoose@5.13.22/node_modules/mongoose/lib/drivers/node-mongodb-native/collection.js:198:23)
gathio-1        |     at listOnTimeout (node:internal/timers:569:17)
gathio-1        |     at process.processTimers (node:internal/timers:512:7)
gathio-1        | 
gathio-1        | Node.js v18.20.4
gathio-1        |  ELIFECYCLE  Command failed with exit code 1.
gathio-1 exited with code 1
 
---

version: "3"

volumes:
    mongodb_data_db:

services:
    gathio:
        build: .
        links:
            - gathio-mongo
        ports:
            - 3000:3000
    gathio-mongo:
        image: mongo:4.4.6
        volumes:
            - ./mongodb_data_db:/data/db

---

[general]
# Your domain goes here. If there is a port it should be 'domain.com:port', but
# otherwise just 'domain.com'.
domain = "localhost:3000"
port = "3000"
email = "***"
site_name = "Gathio"
is_federated = true
# Events will be deleted this many days after they have ended. Set to 0 to
# disable automatic deletion (old events will never be deleted).
delete_after_days = 10
# If left blank, this defaults to
# https://yourdomain.com/images/gathio-email-logo.gif. Set a full URL here to
# change it to your own logo (or just change the file itself).
email_logo_url = ""
# Show a Ko-Fi box to donate money to Raphael (Gathio's creator) on the front
# page.
show_kofi = true
# Show a list of events and groups on the front page which have been marked as
# 'Display this event/group on the public event/group list'.
show_public_event_list = true
# Which mail service to use to send emails to hosts and attendees. Options are
# 'nodemailer' or 'sendgrid'. Configure settings for this mail
# service below.
mail_service = "nodemailer"
# An array of email addresses which are permitted to create events. If this is
# empty, anyone can create events.
# For example:
# creator_email_addresses = ["test@test.com", "admin@test.com"]
creator_email_addresses = []

[database]
# Set up for a locally running MongoDB connection. Change this to
# 'mongodb://mongo:27017/gathio' for a Dockerised connection.
mongodb_url = "mongodb://localhost:27017/gathio"

[nodemailer]
smtp_server = "***"
smtp_port = "465"
smtp_username = "***"
smtp_password = "***"

[sendgrid]
api_key = ""

# Links to static pages (for example a privacy policy) or an external community page,
# which will be displayed in the footer.
# If paths begin with a slash, they are treated as internal and will open the specified
# Markdown or text file. If they are absolute (begin with https://), they will simply
# link to the specified URL.

# [[static_pages]]
# title = "Privacy Policy"
# path = "/privacy"
# filename = "privacy-policy.md"

# [[static_pages]]
# title = "External Link"
# path = "https://example.com"

**Comment by HybridSarcasm** [comment]
Please consider re-formatting your problem report. It would also be helpful for you to provide a summary of the problem.

**Comment by HybridSarcasm** [comment]
But, I forged ahead and read through that mess. I think I found your problem.
Your docker-compose.yml refers to your mongo service as 'gathio-mongo". That means you want to change yuor monodb_url in the config.example.toml file to 'gathio-mongo'. After the change, the parameter should look like this:

`mongodb_url = "mongodb://gathio-mongo:27017/gathio"`

Using 'localhost' was telling the gathio app to look at its own container for mongo, which isn't correct. Mongo is being run in a separate container.
